docs/integrations/sources/delighted.md

```diff
@@ -1,32 +1,52 @@
 # Delighted
 
-This page contains the setup guide and reference information for the Delighted source connector.
+This page contains the setup guide and reference information for the [Delighted](https://delighted.com/) source connector.
 
 ## Prerequisites
+- A Delighted API Key.
+- A desired start date and time. Only data added on and after this point will be replicated.
 
-To set up the Delighted source connector, you'll need the [Delighted API key](https://app.delighted.com/docs/api#authentication).
+## Setup guide
+### Obtain a Delighted API Key
 
-## Set up the Delighted connector in Airbyte
+To set up the Delighted source connector, you'll need a Delighted API key. For detailed instructions, please refer to the 
+[official Delighted documentation](https://app.delighted.com/docs/api).
 
-1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
-2. Click **Sources** and then click **+ New source**.
-3. On the Set up the source page, enter the name for the Delighted connector and select **Delighted** from the Source type dropdown.
-4. For **Since**, enter the date in a Unix Timestamp format. The data added on and after this date will be replicated.
-5. For **API Key**, enter your [Delighted `API Key`](https://delighted.com/account/api).
-6. Click **Set up source**.
+### Set up the Delighted connector in Airbyte
+
+1. [Log in to your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account, or navigate to the Airbyte Open Source dashboard.
+2. In the left navigation bar, click **Sources**.
+
+:::tip
+If this is your first time setting up an Airbyte source, skip the next step and proceed to step 4.
+:::
+
+3. In the top-right corner, click **+ New source**.
+4. Find and select **Delighted** from the list of available sources.
+5. Enter a **Source name** of your choosing.
+6. Enter your **Delighted API Key**.
+7. In the **Date Since** field, enter the desired UTC date and time in an 
+[ISO 8601 format](https://www.iso.org/iso-8601-date-and-time-format.html). Only the data added on and after this date will be replicated.
```

Comment:
I think we don't need to tell them this since the UI already shows a date-picker

Reply:
Removed, and I truncated steps 1-4 as suggested 👍